ensembl_website/migrations/0005_auto_20201119_1644.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.13 on 2020-11-19 16:44
```

Comment:
Not sure this should be part of the PR.

Reply:
See previous message about migrations.

Reply:
Simply remove the migration from your commit / PR please.